src/robots-pipe.js

```diff
@@ -119,9 +120,8 @@ function getForwardedHosts(req) {
 async function computeSurrogateKeys(state) {
   const keys = [];
 
-  const pathKey = `${state.ref}--${state.repo}--${state.owner}${state.info.path}`;
   keys.push(await computeSurrogateKey(`${state.site}--${state.org}_config.json`));
-  keys.push(await computeSurrogateKey(pathKey));
+  keys.push(await computeCodePathKey(state));
```

Comment:
`robots` is always in `code`, that's why you don't need to check the `state.content.sourceBus` here, right?

Reply:
no, robots is always in config

Reply:
but I don't think it matters. I don't know why we added the non config key

Reply:
It's your code ;) I only consolidated it.

Reply:
robots.txt was delivered with the code key before. I only consolidated the code, i.e. use a dedicated function to compute code key.